.rubocop.yml

```diff
@@ -0,0 +1,67 @@
+AllCops:
```

Comment:
Would be nice if there were a consistent RuboCop style across all of the Celluloid projects

Perhaps another one for @digitalextremist's https://github.com/celluloid/culture/issues/2